app/views/pages/guidance.html.erb

```diff
@@ -1,5 +1,9 @@
 <%= content_for :page_title, "Guidance" %>
 
+<% content_for :before_content do %>
+  <%= govuk_back_link_to(:back) %>
```

Comment:
What if the `referrer` is nil?

I think it should take them back to `provider#index` which will force them to sign in...

Reply:
If it's nil I think it defaults to `history.back`

Reply:
Gonna test this now.

Reply:
What happens to `providers_path` if the user isn't logged in?

Reply:
The get taken to DfE-Sign

Reply:
What happens if the user has only 1 organisation?

Reply:
They are re-directed to the `org#show` view